release-notes/major11/11.2.0/valtimo-backend-libraries.md

```diff
@@ -52,6 +52,14 @@ The following features were added:
   Added a new process bean that can resolve and handle values from several sources. More information can be
   found [here](/reference/process-beans.md#valueresolverdelegateservice)
 
+* **Added support for Keycloak group roles**
+
+  Valtimo access control can now make use of Keycloak group roles.
+
+* **Fix: PBAC mapping CamundaTask to JsonSchemaDocument**
```

Comment:
Is this a bugfix or a feature? And I think we should use access control rather than PBAC as a name.